frontend/utility/MultitrackVideoOutput.cpp

```diff
@@ -17,6 +17,9 @@
 
 #include <cinttypes>
 
+// Maximum reconnect attempts with an invalid key error before giving up (roughly 30 seconds with default start value).
```

Comment:
This isn't a sentence. Either drop the final punctuation, or rewrite it to be a complete sentence.